src/main/java/uk/gov/hmcts/reform/unspec/aspect/EventEmitterAspect.java

```diff
@@ -20,15 +19,14 @@
 @RequiredArgsConstructor
 public class EventEmitterAspect {
 
-    private final CaseDetailsConverter caseDetailsConverter;
     private final EventEmitterService eventEmitterService;
 
     @Around("execution(* *(*)) && @annotation(EventEmitter) && args(callbackParams))")
     public Object emitBusinessProcessEvent(ProceedingJoinPoint joinPoint, CallbackParams callbackParams)
         throws Throwable {
         if (callbackParams.getType() == SUBMITTED) {
-            CaseData caseData = caseDetailsConverter.toCaseData(callbackParams.getRequest().getCaseDetails());
-            if (caseData.getBusinessProcess().getStatus() == READY) {
+            CaseData caseData = callbackParams.getCaseData();
+            if (caseData.getBusinessProcess() != null && caseData.getBusinessProcess().getStatus() == READY) {
```

Comment:
would prefer this logic contained in BusinessProcess but not really a part of your PR so fine to leave I guess!

Reply:
added null safety to the existing condition.